content/messaging/agent-chat-api/index.mdx

```diff
@@ -1589,6 +1589,8 @@ It's possible to write to a chat without joining it. The user sending an event w
 
 The method updates the requester's `events_seen_up_to` as if they've seen all chat events.
 
+To set correct agent/bot name use `X-Author-Id` header.
```

Comment:
Why are agents mentioned? Isn't this header needed only when we want to send a message as a bot?

Reply:
It can be used with agents name as far as I know - so we can send event as agent (but I need to confirm that)

Reply:
Ok, please confirm that :) It seems confusing to me as we mention: 
![image](https://user-images.githubusercontent.com/42515529/114384699-5befc180-9b8f-11eb-9e40-233fd1a7b4c6.png)

Reply:
yep, you were right